### ISSUE TITLE
Kind-of-Diploma for German Upper Secondary School Diploma (see also #44)

### ISSUE THREAD

**Issue by GBacharach** [emrex-eu/elmo-schemas#64]
Note: This is a requirement that is part of Issue #44 and is created as a separate issue for ease of processing.

For upper secondary school diplomas (in German “Hochschulzugangsberechtigungen”, short HZB) Germany needs the diploma learning-opportunity’s kind-of-diploma to verify whether this certificate qualifies the applicant to study the study program he applies for. E.g.

Unfortunately EQF-levels are not enough for our purposes. Especially for diplomas within EQF 4 we need more differentiations e.g.

- “Allgemeine Hochschulreife”                  The owner has the right to study (in Germany) whatever and wherever he likes
- “Fachhochschulreife”                              The owner has the right only to study at universities of applied sciences (colleges)
- “Fachgebundene Hochschulreife”          The owner has the right only to study special study programs
- …

It is to be clarified, how the different German upper secondary school education certificates can be mapped in the ELMO data structure. We do not have precisely this differentiation in EQF level 4, even at the German national level (see https://www.cedefop.europa.eu/files/germany_-_european_inventory_on_nqf_2018.pdf page 13):

- Upper secondary general education school leaving certificate (Allgemeine Hochschulreife (AHR))
- Qualification entitling holder to study particular subjects at a higher education institution (Fachgebundene Hochschulreife (FgbHR))
- Qualification entitling holder to study at a university of applied sciences (Fachhochschulreife (FHR))
- Dual VET (three-year and three-and-a-half-year training courses)
- Full-time vocational school (regulated under Länder law) (Berufsfachschule)
- Full vocational qualification (full-time vocational school) (Berufsfachschule)

Germany users need an extension of the ELMO standard here so that we can automatically assign the digital HZBs. Or is there another way to get the information out of ELMO structure for German users?

By the way: ELM (or EDCI) have the same problem (EQF or NQF as attribute of "qualification").

**Comment by mirkostanic** [comment]
Message for EMREX executive committee: 
This is a tricky request. One can not help but ask, why was this structure not communicated to the EQF when it was being made? ELMO currently supports only EQF, if we were to introduce national mappings it would bloat quickly and also there is the issue of readability. Can a non German institution discern between German national levels? It would defeat the point of EQF.

**Comment by GBacharach** [comment]
No idea why the differentiation was not put into EQF. But without that differentiation ELMO cannot be used for German upper secondary school certificates at least not for national purposes (I do not know whether it is needed for international recognitions, but for national recognitions it is needed definitely).

In coordination with the EMREX EC, I therefore propose a solution that optionally supplements such a "Kind-of-Certificate" for Germany.

Additional proposal: In order to be compatible with the German solution, such a supplement could be designed analogously to the "Type of HZB" in the German format XSchule (possibly with the value list that is also used there). More details (unfortunately only in German) at https://xschule.digital/web/spezifikation and in Github.

Please make a proposal for a decision on how this can be implemented in ELMO.

**Comment by mirkostanic** [comment]
This can be solved by supplyinThis can be solved by supplying two "level" elements. 
Example:
```xml
All consumers of ELMO will read this
<level>
    <type>EQF</type>
    <description>European Qualification Framework</description>
    <value>4</value>
</level>
This level will be ignored by everyone except German clients.
<level>
    <type>HZB</type>
    <description>Upper secondary general education school leaving certificate</description>
    <value>Allgemeine Hochschulreife</value>
</level>
```

**Comment by mirkostanic** [comment]
Implemented in V1.7
